packages/components/stories/VirtualizedList.js

```diff
@@ -570,6 +570,33 @@ storiesOf('VirtualizedList', module)
 			</section>
 		</div>
 	))
+	.add('List > Table : desactivation', () => (
+		<div className="virtualized-list">
+			<h1>Virtualized List</h1>
+			<p>
+				Row can be disabled by passing <b>isDisabled</b> function
+				that returns if a row is disabled.
+				<br />
+				Here example <pre>{'isDisabled={item => item.id === 6}'}</pre>
+			</p>
+			<IconsProvider defaultIcons={icons} />
+			<section style={{ height: '50vh' }}>
+				<VirtualizedList
+					collection={collection}
+					id={'my-list'}
+					onRowClick={action('onRowClick')}
+					getAvaibilityStatus={item => (item.id === 2 ? { status: false, message: 'Houlala' } : null)}
```

Comment:
What about 
```suggestion
					getUnavaibilityMessage={row => (row.id === 2 ? { message: 'Houlala' } : null)}
```
which means that it not available